src/repos/projects/Directory.Build.props

```diff
@@ -6,6 +6,10 @@
     <TargetFramework>$(NetCurrent)</TargetFramework>
     <RepositoryName>$(MSBuildProjectName)</RepositoryName>
 
+    <SrcReposDir>$(SrcDir)repos/</SrcReposDir>
+    <KeysDir>$(SrcReposDir)keys/</KeysDir>
+    <SubmoduleDirectory>$(SrcReposDir)src/</SubmoduleDirectory>
```

Comment:
There is an opportunity to make the names more consistent - e.g. Dir vs Directory.  This would expand the scope beyond  the core intent.